### ISSUE TITLE
Ionic Package Cloud Service - Error: Variable(s) missing

### ISSUE THREAD

**Issue by pvdyck** [sarriaroman/FabricPlugin#76]
[Ionic package cloud service 
](http://docs.ionic.io/services/package/)does not work with the plugin for android builds.

The error returned is :
Error: Variable(s) missing (use: --variable FABRIC_API_KEY=value --variable FABRIC_API_SECRET=value).

It may be linked to the fact that custom hooks are not run during ionic cloud builds.

But after looking at the content of the hooks it does not look like it.

Could you please confirm that the hooks only remove things from the plugin installation phase and adds the dSym and such ?

Thanks !

**Comment by Nightbr** [comment]
Hi, 
same problem here (only for Android):

```
:compileReleaseJavaWithJavac/home/package/workspace/apps-20175d24-98/cordova/platforms/android/src/com/sarriaroman/fabric/FabricPlugin.java:142: error: cannot find symbol
						JavaScriptException ex = new JavaScriptException(data.getString(0));
						^
  symbol: class JavaScriptException
/home/package/workspace/apps-20175d24-98/cordova/platforms/android/src/com/sarriaroman/fabric/FabricPlugin.java:142: error: cannot find symbol
						JavaScriptException ex = new JavaScriptException(data.getString(0));
						                             ^
  symbol: class JavaScriptException
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors
 FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

Any idea ?

**Comment by Nightbr** [comment]
Related to issue #75

**Comment by sarriaroman** [comment]
Solved in version 1.1.3
